docs/UserGuide.md

```diff
@@ -43,9 +43,11 @@ title: User Guide
     
 1. Adding a module
 
-    * Allows user to specify module description
+    * Allows user to add any Module of choice into GradPad for tracking purposes.
```

Comment:
Perhaps can specify that modules are added into the Current Modules section/column instead of just GradPad

Reply:
Okay, updated the description! Thanks for feedback.